src/legacy/core_plugins/kibana/public/context/components/action_bar/_action_bar.scss

```diff
@@ -2,13 +2,12 @@
  * 1. Hide increment and decrement buttons for type="number" input.
  */
 .cxtSizePicker {
-  appearance: textfield;
   text-align: center;
   width: $euiSize * 5;
 
   &::-webkit-outer-spin-button,
   &::-webkit-inner-spin-button {
-    appearance: none; /* 1 */
```

Comment:
Nit: These comments are there on purpose and refer to line `2`. I think it's fine the leave them in.

Reply:
... or we add the comment to the line, so I can't miss this connection the next time :)
```
appearance: none; // Hide increment and decrement buttons for type="number" input.
```